dpdd.tex

```diff
@@ -1068,6 +1091,84 @@ \subsubsection{\ForcedSource Table}
+
+radec & double[2] & degrees & Position of the object (centroid), measured jointly across all bands. \\
+
+maskFractionObj & float & ~ & Weighted fraction of this object's pixels that were masked, over all visit images input to the coadd. \\
+
+maskFractionCell & float & ~ & Weighted fraction the coadd cell's that were masked, over all visit images input to the coadd.\footnote{This and \texttt{nEpochCell} are the same for all rows with the same \texttt{cellId}, but are expected to be access per-object; they may or may not be normalized into a separate table in the physical schema.}. \\
+
+nEpochCell & int32[ugrizy] & ~ & Number of input visits that contributed to this coadd cell. \\
+
+g1 & float & ~ & Ellipticity from Gaussian-weighted moments, on-axis, measured consistently over central\footnote{Probably $riz$} bands. \\
+
+g2 & float & ~ & Ellipticity from Gaussian-weighted moments, off-axis. \\
+
+gCov & float[3] & ~ & Covariance matrix for (\texttt{g1}, \texttt{g2}). \\
+
+T & float & arcsec$^{2}$ & Gaussian-weighted moments trace (Ixx + Ixy) \\
+
+SNR & float & ~ & Gaussian weighted signal-to-noise ratio. \\
+
+TErr & float & arcsec$^{2}$ & Uncertainty on \texttt{T}.\\
+
+g1PSFMeta & float & ~ & PSF model ellipticity from Gaussian-weighted moments, on-axis, computed from the central-band-averaged metacalibration-modified PSF models.\footnote{This and other PSF quantities are not constant over a cell only because the PSF may depend on the SED of the object.} \\
+
+g2PSFMeta & float & ~ & PSF model ellipticity from Gaussian-weighted moments, off-axis, computed from the central-band-averaged metacalibration-modified PSF models. \\
+
+TPSFMeta & float & arcsec$^{2}$ & Gaussian-weighted moments trace (Ixx + Ixy) of the central-band-averaged metacalibration-modified PSF models. \\
+
+g1PSFOrig & float & ~ & PSF model ellipticity from Gaussian-weighted moments, on-axis, computed from the central-band-averaged original PSF models. \\
+
+g2PSFOrig & float & ~ & PSF model ellipticity from Gaussian-weighted moments, off-axis, computed from the central-band-averaged original PSF models. \\
+
+TPSFOrig & float & arcsec$^{2}$ & Gaussian-weighted moments trace (Ixx + Ixy) of the central-band-averaged original PSF models. \\
+
+stdFlux & float[ugrizy] & nJy & PSF-corrected aperture fluxes, measured using an approach similar to the ``standard color'' in the main \Object table.\footnote{This is not expected to be a good estimate of total flux, but \emph{some} estimate of total flux is useful, so this is still reported as fluxes, as opposed to the colors in \Object's \texttt{stdColor}.} \\
+
+stdFluxErr & float[ugrizy] & nJy & Uncertainty on \texttt{stdFlux}. \\
+
+stdFluxT & float[ugrizy] & arcsec$^{2}$ & Moments trace (Ixx + Ixy) computed with the same weights as \texttt{stdFlux}. \\
+
+stdFluxTErr & float[ugrizy] & arcsec$^{2}$ & Uncertainty on \texttt{stdFluxT}. \\
+
+flags & bit[8] & bit & Various useful flags. \\
```

Comment:
How wide is this flags column in bits? Is it 8 bytes, so 64 bits or only 8 bits?